API/API.js

```diff
@@ -0,0 +1,100 @@
+export const fetcher = {
+  allTimers() {
```

Comment:
This looks amazing! Great naming conventions!

Reply:
This looks really good. I like the amount of control that this allows for bringing in data